### PR TITLE
Enable periodic tests for release-1.9 branch

### PR DESCRIPTION
**Pull request by SgtCoDFish**
This is part of the [release process](https://cert-manager.io/docs/contributing/release-process/):

> (initial alpha only) Create a PR on [jetstack/testing](https://github.com/jetstack/testing), changing the release-next periodic tests interval to 2h, configured in

### CODE REVIEW

File: config/jobs/cert-manager/cert-manager/release-next/cert-manager-release-next-periodics.yaml [hunk 1]
```diff
@@ -1,8 +1,347 @@
...
+  annotations:
+    testgrid-create-test-group: 'true'
+    testgrid-dashboards: jetstack-cert-manager-next
+    testgrid-alert-email: cert-manager-dev-alerts@googlegroups.com
+    description: Runs the end-to-end test suite against a Kubernetes v1.22 cluster
+  labels:
+    preset-service-account: "true"
+    preset-dind-enabled: "true"
+    preset-cloudflare-credentials: "true"
+    preset-enable-all-feature-gates-disable-ssa: "true"
```
Comment (irbekrm) [inline config/jobs/cert-manager/cert-manager/release-next/cert-manager-release-next-periodics.yaml:150]: For 1.22 we do want to run SSA tests

**Review by irbekrm — CHANGES_REQUESTED** [review]
Thanks @SgtCoDFish !

I've added  nits for running with SSA enabled on kube 1.22+ apart from that happy to lgtm

**Comment by SgtCoDFish** [comment]
Thanks @irbekrm good catch! I've added a commit enabling SSA for those tests :grin:

**Comment by irbekrm** [comment]
Thanks @SgtCoDFish !

/lgtm
/approve

**Pull request merged**